docs/user_guide/SIMP_Administration/Classification_and_Data/SIMP_Compliance_Engine.inc

```diff
@@ -0,0 +1,130 @@
+.. _SIMP Compliance Engine:
+
+The SIMP Compliance Engine
+--------------------------
+
+The :term:`SIMP Compliance Engine` allows users to enable both Puppet
+:term:`class parameter` enforcement as well as code-level validation of both
+class and defined type parameters in compliance with regulatory and internal
```

Comment:
Consider replacing `defined type` with `defined-type` for clarity.  Otherwise, it's easy to mis-read `class` and `defined` as two different kinds of `type parameter`.